audio/bitwig-studio/bitwig-studio.info

```diff
@@ -1,10 +1,10 @@
 PRGNAM="bitwig-studio"
-VERSION="5.1.7"
+VERSION="5.1.9"
 HOMEPAGE="https://www.bitwig.com/"
 DOWNLOAD="UNSUPPORTED"
 MD5SUM=""
-DOWNLOAD_x86_64="https://downloads.bitwig.com/stable/5.1.7/bitwig-studio-5.1.7.deb"
-MD5SUM_x86_64="9f5bf6973ed966a8b95f322d9f558a68"
+DOWNLOAD_x86_64="https://www.bitwig.com/dl/Bitwig%20Studio/5.1.9/installer_linux/bitwig-studio-5.1.9.deb"
```

Comment:
CI failed to download this but i can download this manually. Can you check it again?